File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-content-controls-to-word-documents_2.vb
    Dim richTextControl2 As Microsoft.Office.Tools.Word.RichTextContentControl

    Private Sub AddRichTextControlAtRange()
        Me.Paragraphs(1).Range.InsertParagraphBefore()
        richTextControl2 = Me.Controls.AddRichTextContentControl(Me.Paragraphs(1).Range, _
            "richTextControl2")
        richTextControl2.PlaceholderText = "Enter your first name"
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-content-controls-to-word-documents_2.cs
        private Microsoft.Office.Tools.Word.RichTextContentControl richTextControl2;

        private void AddRichTextControlAtRange()
        {
            this.Paragraphs[1].Range.InsertParagraphBefore();

            richTextControl2 = this.Controls.AddRichTextContentControl(this.Paragraphs[1].Range,
                "richTextControl2");
            richTextControl2.PlaceholderText = "Enter your first name";
        }